.github/workflows/build.yml

```diff
@@ -216,23 +216,6 @@ jobs:
 
       - name: Run tests via docker
         run: cd sui && make test-docker
-
-  terra:
-    runs-on: ubuntu-24.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
-        with:
-          node-version: "22"
-      - run: cd terra && make test
-  terra-2:
```

Comment:
I'm not terra-2 should be removed. this may be a bit of a misnomer, but may actually test wormchain functionality as it is the same deployment for all cosmwasm chains.